res/templates/section_wrapper.html

```diff
@@ -1,4 +1,4 @@
-<details data-level="2" open>
+<details data-level="{{ section_index }}" open>
     <summary class='section-heading'><h2 id="{{ section_anchor }}">{% autoescape false %}{{ section_line }}{% endautoescape %}</h2></summary>
```

Comment:
I think this can not always be `h2`, it should be `h1`, `h3`.. depending of the level right?